templates/deployment.yaml

```diff
@@ -14,14 +14,16 @@ spec:
       labels:
         app: {{ .Release.Name }}-helper
     spec:
+      nodeSelector:
+        releaseName: {{ .Release.Name }}
       imagePullSecrets:
         - name: {{ .Release.Name }}-ghcr
       containers:
         - name: {{ .Release.Name }}-helper
           image: {{ .Values.helperImage }}
           imagePullPolicy: Always
           command: ["ipa-helper"]
-          args: ["--identity", "1",
+          args: ["--identity", "{{ trimPrefix "h" .Release.Name }}",
```

Comment:
why not just having the `releaseName` be the integer?

Reply:
I thought about it, but decided against it. "Names" don't usually start with a number and I'm using the releaseName in a bunch of places that expect valid variable names.